### ISSUE TITLE
Any updates on Sierra 10.12.6 wireless compatibility

### ISSUE THREAD

**Issue by bfeinstein** [360Controller/360Controller#504]
I am unable to get the driver to recognize my wireless 360 controller. Any advice for me?

**Comment by LiamBer** [comment]
same here

**Comment by PKBeam** [comment]
I'm having issues with a wireless 360 controller, on High Sierra. My friend running Yosemite doesn't have these issues.

**Comment by FranticRain** [comment]
Before we can help you, please consult the README and provide all necessary information so that we can help you with your issue. https://github.com/360Controller/360Controller#wired-xbox-360-controllers

**Comment by FranticRain** [comment]
This issue is being closed due to inactivity. We can't fix issues if we don't have enough information. So periodically we close down issues that can't be resolved if they seem to have been abandoned by their creators. If your issue still persists, please open a new issue after thoroughly consulting the README and providing all information as required by the README.
